Customer Demo/Customer Demo/Screens/SignIn/SignInViewController.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  SignInViewController.swift
+//  Customer Demo
+//
+//  Copyright 2022 Ably Real-time Ltd (ably.com)
+//
+
+import Foundation
```

Comment:
`Foundation` import is redundant if you're already importing `UIKit`, can be removed